src/store/thunks/posts.js

```diff
@@ -0,0 +1,15 @@
+import { getAllPosts } from "store/actions/posts";
+import { db } from "database";
+
+export const getAllPostsToRender = () => (dispatch) => {
```

Comment:
A more apt name would be `getPosts`. The function does not care whether you are going to render the posts. It's singular job is to fetch you the posts.

Reply:
Changed to fetchPosts along with the rest of the things